.github/workflows/ansible-test.yml

```
@@ -42,8 +42,16 @@ jobs:
       - name: Install ansible-base
         run: pip install git+https://github.com/ansible-collection-migration/ansible-base.git --disable-pip-version-check
 
+      - name: Install community.general
+        run: ansible-galaxy collection install community.general
```

Ah, so it appears that https://github.com/ansible-collections/mongodb/pull/8/files#diff-237eb72aa3cd64f24dd73db02357b18dR35 causes the GitHub Action to chdir into `ansible_collections/community/mongodb`, so we just need to go up two directories

```suggestion
      - name: Install community.general
        run: ansible-galaxy collection install community.general -p ../../
```

We can tell this works from the ansible-test log where it says
```
quay.io/ansible/default-test-container:1.12
Scanning collection root: /home/runner/work/mongodb/ansible_collections
Including collection: community.general (6204 files)
Including collection: community.mongodb (59 files)
Creating a payload archive containing 7035 files...
```

---

Great OK. I was wondering if ansible-test is smart enough to include the modules in the current branch? For example if I use mongodb_replicaset will ansible-test call the one in the current branch like the former ansible architecture? i.e. I don't have to install the mongodb collection as I do with community.general. Otherwise we have a chicken vs egg situation.